### PR TITLE
[JBPM-9880] logging async node enter

### PR DESCRIPTION
**Pull request by elguardian**
Jira: https://issues.redhat.com/browse/JBPM-9880

### CODE REVIEW

**Review by afalhambra — COMMENTED** [review]
Very minor comments, mostly cosmetics. Great work @elguardian

File: kie-api/src/main/java/org/kie/api/event/process/ProcessAsyncNodeScheduledEvent.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2005 Red Hat, Inc. and/or its affiliates.
```
[the remark this answers is not in the document]
Comment (elguardian) [reply kie-api/src/main/java/org/kie/api/event/process/ProcessAsyncNodeScheduledEvent.java:2]: done

File: kie-api/src/main/java/org/kie/api/event/process/ProcessNodeEvent.java [hunk 2]
```diff
@@ -32,4 +35,11 @@
      */
     NodeInstance getNodeInstance();
 
+    /**
+     * this allow to add some extra data across node events
```
[the remark this answers is not in the document]
Comment (elguardian) [reply kie-api/src/main/java/org/kie/api/event/process/ProcessNodeEvent.java:39]: done

File: kie-api/src/main/java/org/kie/api/event/process/ProcessEventListener.java [hunk 3]
```diff
@@ -114,4 +114,11 @@ default void onSignal(SignalEvent event) {}
      * @param event
      */
     default void onMessage(MessageEvent event) {}
+
+    /**
+     * this listener method is invoked when async node is entered triggering
+     * a scheduled job for executiong this node.
```
[the remark this answers is not in the document]
Comment (elguardian) [reply kie-api/src/main/java/org/kie/api/event/process/ProcessEventListener.java:120]: done

File: kie-api/src/main/java/org/kie/api/event/process/ProcessEventListener.java [hunk 4]
```diff
@@ -114,4 +114,11 @@ default void onSignal(SignalEvent event) {}
...
+     * @param event
+     */
+    default void onAsyncNodeEventScheduled(ProcessAsyncNodeScheduledEvent event) {}
```
[the remark this answers is not in the document]
Comment (elguardian) [reply kie-api/src/main/java/org/kie/api/event/process/ProcessEventListener.java:123]: done

**Review by afalhambra — APPROVED** [review]
nice one!

**Pull request closed without merging**